src/components/ShowHide/ShowHide.tsx

```diff
@@ -100,7 +100,7 @@ export const ShowHide = ({
    * 1) If escape button is struck, close the show hide contents
    */
   function handleOnKeyDown(e: KeyboardEvent<HTMLElement>) {
-    if (e.code === ESCAPE_KEYCODE) {
+    if (e.key === ESCAPE_KEYCODE) {
```

Comment:
> doesn't work in our environment

Does this mean it didn't work in tests (maybe jsdom doesn't implement `code`)? Or did it not work in the browser (maybe React synthetic events don't implement it)?

In any case, I think `event.key` may be what we want anyway. If I'm reading MDN right:
- `code` represents the physical key, not the resulting character that key represents.
- `key` represents the character